deploy/default/kustomization.yaml

```diff
@@ -1,17 +0,0 @@
-apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
actually, this one we probably should keep, as an easy way to deploy BMO standalone, you should change the ironic here for BMO standalone

Reply:
I think I should have it and also with same content as I have it in [PR#522](https://github.com/metal3-io/baremetal-operator/blob/d881359c2920f1b992cd06667fa990f0177e2030/deploy/default/kustomization.yaml) where I am modifying this file. Because that one will be merged first, and when we will merge this it will actually delete it, right?

Reply:
Well you would most probably get a conflict, so let's focus on the other one first